Packs/Pcysys/Scripts/PenteraOperationToIncident/PenteraOperationToIncident.yml

```diff
@@ -0,0 +1,27 @@
+commonfields:
+  id: PenteraOperationToIncident
+  version: -1
+name: PenteraOperationToIncident
+script: ''
+type: python
+tags: []
+enabled: true
+args:
+  - name: full_action_report
+    required: true
+    description: Pentera Full Action Report
```

Comment:
can you give an example? should that be retrieved from the integration? from which command?

Reply:
It is the integration's output when finished running a task. Produced by the command `pentera-get-task-run-full-action-report`

Reply:
please add that to the argument description